### PR TITLE
Fallback points for bounding boxes

### PR DESCRIPTION
**Pull request by einarf**
The new hit box system is a lot more sensitive to missing hit box points. In 2.6 we manually returned a bounding box around the image in the sprite methods if points was not present. This PR restores the old behavior, but we might instead want to handle missing points differently in 3.0?

### CODE REVIEW

File: arcade/hitbox/base.py [hunk 1]
```diff
@@ -65,6 +65,24 @@ def __call__(self, *args: Any, **kwds: Any) -> Self:
         """
         return self.__class__(*args, **kwds)  # type: ignore
 
+    def create_bounding_box(self, image: Image) -> PointList:
+        """
+        Crate points for a simple bounding box around an image.
```
Comment (gran4) [inline arcade/hitbox/base.py]: Create is misspelled

File: tests/unit/htibox/test_black_image.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    )
+    assert tex.hit_box_points == EXPECTED
+
+
+def test_pymunk():
+    tex = arcade.Texture(
+        Image.new("RGBA", (100, 100)),
+        hit_box_algorithm=arcade.hitbox.algo_detailed,
+    )
+    assert tex.hit_box_points == EXPECTED
```
Comment (gran4) [inline tests/unit/htibox/test_black_image.py:23]: Wouldn't you want to add a check for when there is no texture?

**Review by gran4 — CHANGES_REQUESTED** [review]

**Comment by gran4** [comment]
I just realized this is a draft... Sorry

**Comment by einarf** [comment]
> I just realized this is a draft... Sorry

All good! Reviewing things never hurts and I might have missed the typo.

**Pull request merged**